handlers/modchat.py

```diff
@@ -19,6 +19,7 @@ async def get_roominfo(conn: Connection, room: Room, *args: str) -> None:
 
     if args[0] == "chat":
         await conn.send(f"|/cmd roominfo {room.roomid}")
+        await room.send("/roomlanguage", False)
```

Comment:
A detail I didn't mention in the PR description.

This handler contains actions done upon joining a room. Until now, these were only needed for modchat purposes, but since we also detect the language now, should we just move it to `handlers/room.py`?

Alternatively I could create a second handler just for `/roomlanguage` but it seems repetitive and probably more chaotic.